server/train/pipelines/XGBoostRegressionCompDynPipeline/pipe.py

```diff
@@ -2,40 +2,47 @@
 import sys
 from typing import List, Tuple, Dict, Any
 import pandas as pd
+import copy
 
 train_path = os.path.join(os.path.dirname(__file__), '../../')
 prom_path = os.path.join(os.path.dirname(__file__), '../../../prom')
 sys.path.append(train_path)
 sys.path.append(prom_path)
 
-from pipeline import TrainPipeline
-from train_types import FeatureGroup, FeatureGroups, PACKAGE_LABEL, ModelOutputType, XGBoostRegressionTrainType
+from train_types import FeatureGroup, FeatureGroups, EnergyComponentLabelGroups, EnergyComponentLabelGroup, ModelOutputType, XGBoostRegressionTrainType
 from pipe_util import XGBoostRegressionModelGenerationPipeline
 from prom.query import PrometheusClient
 from extractor import DefaultExtractor
 
+
 # Currently Cgroup Metrics are not exported
-class XGBoostRegressionCompDynPipeline(TrainPipeline):
+class XGBoostRegressionStandalonePipeline():
 
-    def __init__(self, train_type: XGBoostRegressionTrainType, save_location: str) -> None:
-        self.train_type = train_type
-        self.save_location = save_location
-        model_name = XGBoostRegressionCompDynPipeline.__name__ + "_" + self.train_type.name
-        model_file = model_name + ".model"
+    def __init__(self, train_type: XGBoostRegressionTrainType, save_location: str, node_level: bool) -> None:
         self.model = None
-        model_class = 'xgboost'
+        self.train_type = train_type
+        self.model_class = 'xgboost'
         self.energy_source = 'rapl'
-        self.energy_components = ['package', 'dram']
-        features = FeatureGroups[FeatureGroup.CgroupOnly]
-        model_output = ModelOutputType.DynComponentPower
-        self.labels = PACKAGE_LABEL
-        super().__init__(model_name, model_class, model_file, features, model_output)
+        self.feature_group = FeatureGroup.CounterOnly
```

Comment:
also include irq features

Reply:
Included. I should note that in the next PR (which involves updating extractor for kepler metrics), IRQ features will be stored in BPF. Moreover, pipeline won't work yet because the kepler metrics have not been updated in the kepler model server. Will finish this today.